rebasehelper/tests/conftest.py

```diff
@@ -59,3 +63,26 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+def setattr_(instance, name, value):
+    try:
+        if next(iter(inspect.signature(value).parameters), None) == 'self':
+            # bind method to instance
+            value = types.MethodType(value, instance)
+    except TypeError:
+        pass
+    setattr(instance, name, value)
+
+
+@pytest.fixture
+def mocked_spec_object(spec_attributes):
+    spec = SpecFile.__new__(SpecFile)
+    spec.save = lambda: None
```

Comment:
I think you need to use `setattr_` with `lambda self: None` here, otherwise it will create a static method.